scripts/commit-msg.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+PATH=.venv/bin:"$PATH"
+
+git secrets --commit_msg_hook -- "$@"
```

Comment:
I still get
`.venv/bin/git-secrets: line 52: git secrets --aws-provider: command not found`